<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConsoleInterface</class>
 <widget class="QDialog" name="ConsoleInterface">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>447</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Konsola połączenia z portem </string>
  </property>
  <widget class="QLineEdit" name="onlineCommand">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>410</y>
     <width>351</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="communicationConfiguration">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>551</width>
     <height>101</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title">
    <string>Dane połączenia</string>
   </property>
   <widget class="QPushButton" name="configureButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>70</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Konfiguruj</string>
    </property>
   </widget>
   <widget class="QLabel" name="connectionData">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>411</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="sent">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>551</width>
     <height>111</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title">
    <string>Wysłano</string>
   </property>
   <widget class="QListView" name="sentView">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>531</width>
      <height>81</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>100</horstretch>
      <verstretch>100</verstretch>
     </sizepolicy>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="received">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>551</width>
     <height>111</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title">
    <string>Odebrano</string>
   </property>
   <widget class="QListView" name="receivedView">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>531</width>
      <height>81</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>100</horstretch>
      <verstretch>100</verstretch>
     </sizepolicy>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>410</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Zamknij</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>370</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Wyślij</string>
   </property>
  </widget>
  <widget class="QLabel" name="onlineLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>46</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Online</string>
   </property>
  </widget>
  <widget class="QLabel" name="offlineLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Offline</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="offlineCommand">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>370</y>
     <width>351</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="getCommand">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>370</y>
     <width>25</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>ConsoleInterface</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>540</x>
     <y>422</y>
    </hint>
    <hint type="destinationlabel">
     <x>581</x>
     <y>403</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sendButton</sender>
   <signal>clicked()</signal>
   <receiver>ConsoleInterface</receiver>
   <slot>sendMessage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>529</x>
     <y>373</y>
    </hint>
    <hint type="destinationlabel">
     <x>588</x>
     <y>357</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>getCommand</sender>
   <signal>clicked()</signal>
   <receiver>ConsoleInterface</receiver>
   <slot>chooseCommand()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>458</x>
     <y>379</y>
    </hint>
    <hint type="destinationlabel">
     <x>476</x>
     <y>353</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>configureButton</sender>
   <signal>clicked()</signal>
   <receiver>ConsoleInterface</receiver>
   <slot>configureCommunication()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>527</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>485</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>sendMessage()</slot>
  <slot>chooseCommand()</slot>
  <slot>configureCommunication()</slot>
 </slots>
</ui>
